src/status_im/contexts/onboarding/events.cljs

```diff
@@ -156,33 +157,15 @@
  (fn [{:keys [db]} [auth-method]]
    {:db (assoc db :auth-method auth-method)}))
 
-(def ^:const temp-display-name
-  "While creating a profile, we cannot use an empty string; this value works as a
-  placeholder that will be updated later once the compressed key exists. See
-  `status-im.contexts.profile.edit.name.events/get-default-display-name` for more details."
-  "temporal username")
-
-(rf/reg-event-fx
- :onboarding/use-temporary-display-name
- (fn [{:keys [db]} [temporary-display-name?]]
-   {:db (assoc db
-               :onboarding/profile
-               {:temporary-display-name? temporary-display-name?
-                :display-name            (if temporary-display-name?
-                                           temp-display-name
-                                           "")})}))
-
```

Comment:
Removed this and related code because now the `display-name` is allowed to be empty, in that case, the alias is shown. The new alias is the compressed key with ellipsis